src/YawPitchControl/YawPitchControl.js

```diff
@@ -324,7 +307,12 @@ export default class YawPitchControl extends Component {
 		this._togglePinchInputByOption(this.options.touchDirection, this.options.useZoom);
 
 		if (keys.some(key => key === "touchDirection")) {
-			this._enabled && this._enableTouch(this.options.touchDirection);
+			const isVR = this.options.gyroMode === GYRO_MODE.VR;
```

Comment:
@WoodNeck 

There are a lot of this.options and often use duplicated options.
isVR seems to use it from above.